pkg/application/inject/fuse/injector.go

```diff
@@ -364,3 +366,11 @@ func (s *Injector) Inject(in runtime.Object, runtimeInfos map[string]base.Runtim
 func (s *Injector) InjectUnstructured(in *unstructuredtype.Unstructured, runtimeInfos map[string]base.RuntimeInfoInterface) (out *unstructuredtype.Unstructured, err error) {
 	return nil, fmt.Errorf("not implemented yet")
 }
+
+func (s *Injector) getFuseDeviceResourceNameFromEnv() string {
```

Comment:
Why not making it as a global function? It can be used in more scenarios  in future.

Reply:
Done.